spec/factories/imparted_hour_factory.rb

```diff
@@ -5,7 +5,7 @@
     journey
     default_minutes = Random.rand(1..120)
     minutes { default_minutes }
-    date { Date.today }
+    date { Time.zone.today }
```

Comment:
use `Time.current.to_date` instead. Same result, but you should be using `Time.current` for almost everything.

The reason is that 1 thing is easier to memorize than N things